MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/DirectionsActivity.java

```diff
@@ -144,7 +142,7 @@ private void initLayers(@NonNull Style loadedMapStyle) {
    * @param origin      the starting point of the route
```

Comment:
Shall we add java doc for the new parameter?

Reply:
Sure. Added.